README.md

```diff
@@ -7,12 +7,8 @@ Talk to ChatGPT via any Matrix client!
 
 A Matrix bot that uses [waylaidwanderer/node-chatgpt-api](https://github.com/waylaidwanderer/node-chatgpt-api) to access the unofficial ChatGPT API.
 
-## ⚠️ Currently there is no free ChatGPT model provided by the OpenAI-API ⚠️
-
-The bot <ins>won't work</ins> as expected out of the box and there is no known free model which can be used:
-
-- The `CHATGPT_MODEL` environment can be used to set the model - as of writing the default uses a model of ChatGPT from late 2022 which is **not working at the moment** and we do not know if and when this will change. However, you can change the `CHATGPT_MODEL` env to `text-davinci-003` or any other of the [supported models](https://platform.openai.com/docs/models/gpt-3). Keep in mind that that these alternative models are not free and will cost you OpenAI credits.
-- Check the [waylaidwanderer/node-chatgpt-api](https://github.com/waylaidwanderer/node-chatgpt-api) repository for any updates on this.
+## Warning for users upgrading from version 2.x
+OpenAI released the [official API for ChatGPT](https://openai.com/blog/introducing-chatgpt-and-whisper-apis). Thus, we no longer have to use any older models or any models which kept on being turned off by OpenAI. This means the bot is now way more stable and way faster. However, please note: The usage of the API is **no longer free**. If you use this bot, your OpenAI account **will be charged**! You might want to limit your budget in your account using the OpenAI website.
```

Comment:
should we link to https://platform.openai.com/account/billing here?

Reply:
Can we catch the warning, the API returns when the limit is reached? Would be helpful if the bot returns a message like "API usage reached..."

Reply:
btw: you missed line 8.  `to access the unofficial ChatGPT API.`. Should be the `official API`.